src/RECONR/ReferenceGrid/test/call.test.cpp

```diff
@@ -272,3 +241,16 @@ std::string ReichMoore(){
     " 6.209400+0 3.000000+0 1.654700-4 4.005100-2-1.079400-1 7.385900-26922 2151     \n";
 }
 
+/*
```

Comment:
Is this test no longer valid or was it accidentally commented out?

Reply:
accidental